templates/vue/src/components/NodeModal.vue

```diff
@@ -528,6 +640,15 @@ export default {
         { name: "fullscreen", value: this.node.fullscreen },
         { name: "tydeType", value: this.node.tydeType },
         { name: "showInBackpack", value: this.node.showInBackpack },
+        { name: "planetViewNotEarnedIconUrl", value: this.node.typeData.planetViewNotEarnedIconUrl },
+        { name: "planetViewEarnedIconUrl", value: this.node.typeData.planetViewEarnedIconUrl },
+        { name: "spaceshipPartNotEarnedIconUrl", value: this.node.typeData.spaceshipPartNotEarnedIconUrl },
+        { name: "spaceShipPartEarnedIconUrl", value: this.node.typeData.spaceShipPartEarnedIconUrl },
+        { name: "spaceShipPartHoverUrl", value: this.node.typeData.spaceShipPartHoverUrl },
```

Comment:
All the other images finish with `IconUrl` but this one ends with just `Url`. Can you please keep this consistent by making it end with `IconUrl` as well?